packages/remix-dev/__tests__/fixtures/replace-remix-imports/.github/workflows/deploy.yml

```diff
@@ -0,0 +1,205 @@
+name: 🚀 Deploy
```

Comment:
Could we remove unnecessary files from the fixtures? I worry that including them will cause confusion and issues in the future when people are doing a big find/replace sort of thing in the codebase and they come across files like this and spend extra time to make sure they make proper changes when it actually doesn't even matter.

So let's include only the files that can justify their existence in the fixture (which is to say that most of the files here should probably be deleted or, at the very least, drastically simplified).